docs/source/azure/advanced/azurefunctions.rst

```diff
@@ -34,7 +44,26 @@ provisioned. Application Insights has six available locations and thus can not a
 region as the other resources: West US 2, East US, North Europe, South Central US, Southeast Asia, and
 West Europe. The sku, skuCode, and workerSize correlate to scaling up the App Service Plan.
 
-An example on how to set the provision-options when running in azure functions mode:
+An example on how to set the servicePlanName and accept defaults for the other values:
+
+.. code-block:: yaml
+
+    policies:
+      - name: stopped-vm
+        mode:
+            type: azure-periodic
+            provision-options:
+              servicePlanName: functionshost
+         resource: azure.vm
+         filters:
```

Comment:
Maybe we should add an action here, until we give clear instructions on how to see the output.

Reply:
Yeah just didn't put any thought into it yet as I haven't ever run an action even, hah.  The docs were mostly for you so far :)   Once it is all working with final naming and stuff we can write a bunch.